.test_runner_config.yaml

```diff
@@ -40,7 +40,8 @@ steps:
     -a ${server_password} --setup-dns --auto-forwarders
   - ipa-kra-install -p ${server_password}
   lint:
-  - make V=0 lint
+  - PYTHON=/usr/bin/python2 make V=0 lint
+  - PYTHON=/usr/bin/python3 make V=0 lint
```

Comment:
I don't think that running those linters twice is an issue, but as you wish I updated PR